src/Ubiquity/orm/creator/Model.php

```diff
@@ -122,8 +122,17 @@ private function getToStringField(){
 		$pwArray=array('AmericanEnglish'=>'password','BrazilianPortuguese'=>'senha','Croatian'=>'lozinka','Czech'=>array('heslotajne','helslo_tajne'),'Danish'=>'password','Dutch'=>'wachtwoord','EuropeanSpanish'=>'contrasena','Finnish'=>'salasana','French'=>array('motdepasse','mot_de_passe'),'German'=>'passwort','Italian'=>'password','Norwegian'=>'passord','Polish'=>'haslo','EuropeanPortuguese'=>'senha','Romanian'=>'parola','Russian'=>'naponb','LatinAmericanSpanish'=>'contrasena','Swedish'=>array('loesenord','losenord'),'Turkish'=>'sifre','Ukrainian'=>'naponb','Vietnamese'=>array('matkhau','mat_khau'));
```

Comment:
Hi @ndL-1990 
I will put a one dimension array, it is not important to know the country corresponding to the password.
And it'll lighten your search algorithm

Reply:
Hi @jcheron ,
Okay I will change it to one dimension array.
So the test will be easier to do.